src/components/Header.js

```diff
@@ -1,97 +1,19 @@
-import React, { useContext } from "react";
+import React from "react";
 import styled from "styled-components";
-import { Link as GatsbyLink } from "gatsby";
-import BootstrapNavbar from "react-bootstrap/Navbar";
-import Nav from "react-bootstrap/Nav";
 
-import { AuthUserContext, withFirebase } from "upe-react-components";
-import {
-  isNonMember,
-  isApplicantOrRecruitmentTeam,
-  isAdmin,
-} from "../util/conditions";
-
-const Navbar = styled(BootstrapNavbar)`
-  background: #333333;
-  border-top: 3px solid ${(props) => props.theme.palette.mainBrand};
-  color: white;
-  padding: 1rem 1rem;
+import Background from "../images/main-background.png";
```

Comment:
I don't think the header should be replaced with an image, the navbar makes for a quicker navigation experience, and the image takes up a lot of space that gets wasted, it would make more sense to just show the masthead on the landing page. Also note that the image doesn't work on all screen sizes, e.g. 
![image](https://user-images.githubusercontent.com/5439867/150693737-b25a8184-a4e6-45a4-979b-7ec5655bec66.png)
The text on top and below the logo are cutoff, so either the image needs to be resized better or just made more simple to just be the logo